README.md

```diff
@@ -1,5 +1,3 @@
-<a name="readme-top"></a>
-
 <!--
```

Comment:
- [ ] Good job trying to set up your readme. For your readme to be professional, kindly ensure that all the sections of the readme are present(only the FAQ section is optional) and the descriptions are properly customized to suit your project.

Also, remember to remove all the placeholder texts which were there to guide you on the content to write in each section ad also remove the microverse logo or replace it with yours.